packages/demo/examples/08-network/renderer/AttributeNode.js

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const proptypes = {
+  nodeStyles: PropTypes.object,
+  node: PropTypes.object.isRequired,
+  onMouseMove: PropTypes.func,
+  onClick: PropTypes.func,
+  onMouseLeave: PropTypes.func,
+  onMouseEnter: PropTypes.func,
+};
+
+const defaultProps = {
+  nodeStyles: {
+    fill: '#15aabf',
+    stroke: 'none',
+    strokeWidth: 0,
+    opacity: 0.5,
+    defaultSize: 3,
+  },
+  onMouseMove: undefined,
+  onClick: undefined,
+  onMouseLeave: undefined,
+  onMouseEnter: undefined,
+};
+
+export default function AttributeNode(props) {
+  const { nodeStyles, node, onMouseMove, onClick, onMouseLeave, onMouseEnter } = props;
+  const { stroke, strokeWidth, fill, opacity, defaultSize } = nodeStyles;
+  return (
+    <g opacity={node.opacity || opacity} >
+      <rect
+        // key={node.id}
+        x={-node.size/2}
+        y={-node.size/2}
+        width={node.size || defaultSize}
+        height={node.size || defaultSize}
+        fill={node.fill || fill}
+        stroke={node.stroke || stroke}
+        strokeWidth={strokeWidth}
+        onMouseMove={onMouseMove && ((event) => {
+          onMouseMove({ event, data: node });
```

Comment:
why do you return the `node` in this case and `index` for the other cases? I think consistency across all event handlers is ideal/more expected. also for convenience could you pass both the node `id` and `index` so the user can look things up easily if they have a map by id?